recipes/geospatial/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "geospatial" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geospatial-{{ version }}.tar.gz
+  sha256: b4a4e45d7b876a468188343cc52bbbfad1b348800fa21e26a8eb891b16dd2f83
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - geemap
+    - geopandas
+    - ipygany
+    - ipyvtklink
+    - keplergl
+    - laspy
+    - leafmap
+    - localtileserver
+    - netcdf4
+    - osmnx
+    - rasterio
+    - panel
+    - psycopg2
+    - pydeck
+    - pyntcloud
+    - pyvista
+    - rio-cogeo
+    - rioxarray
+    - sqlalchemy
+    - streamlit-folium
+    - xarray-spatial
+    - quilt3
+    - httplib2shim
+    - python-box <=5.4
+    - ipyevents <=0.9.0
+    - setuptools >=60
+    - google-api-python-client <=2
```

Comment:
None of these are listed upstream. Are you sure that they are needed? Especially `setuptools` seems strange as that is normally not needed when running the package.

Reply:
The issue was caused by `pip check`. In the past, the recommended recipe template does not include `pip check`. Some of the submitted recipes were accepted because `pip check` was not used at the time even though they would fail should `pip check` be used . Now they are causing issues for the downstream packages because of them due to the use of `pip check`. Therefore, I had to add those missing packages to the end of the recipe in order to pass `pip check`. Anyway, I have removed the packages in question, and it should now resolve the issue.